src/io.rs

```diff
@@ -2,85 +2,105 @@
 
 use core::{fmt, slice};
 use core::fmt::Write;
-
-/// File descriptors
-const STDOUT: usize = 1;
-const STDERR: usize = 2;
+use nr;
 
 /// Host's standard error
-struct Stderr;
+pub struct Stderr {
+    fd: usize,
+}
+
+/// Construct a new handle to the host's standard error.
+pub fn stderr() -> Stderr {
```

Comment:
But `std::io::stdout` doesn't implicitly `panic!`; it does something weirder: it returns a Fake handle who's write methods are all no-ops. I don't think we should do the same.